build/gen_stub.php

```diff
@@ -1640,7 +1640,8 @@ public function initializeZval(string $zvalName, iterable $allConstInfos): strin
         if ($this->type->isNull()) {
             $code .= "\tZVAL_NULL(&$zvalName);\n";
         } elseif ($this->type->isBool()) {
-            $code .= "\tZVAL_BOOL(&$zvalName, " . ($cConstValue ?: ($this->value ? "true" : "false")) . ");\n";
+            $boolType = $this->value ? 'TRUE' : 'FALSE';
+            $code .= "\tZVAL_$boolType(&$zvalName);\n";
```

Comment:
@kocsismate, what do you think?

Reply:
I was OK with the use of `ZVAL_BOOL` in arginfo files as they are just generated code. But I'm ok with this change as well. I'd be more happy though if the whole `ZVAL_FALSE`/`ZVAL_TRUE` was concatenated into the `$code` variable so that searching for the expression becomes easier:
```
$code .= "\t" . ($this-value ? "ZVAL_TRUE" : "ZVAL_FALSE") . "(&$zvalName);\n";
```